.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+<!-- If you are raising a bug playing a stream, you must fill out the following or your issue may not be responded to. For features or improvements, you may delete this. -->
+##### Prerequisites
+- [ ] Have you checked for duplicate [issues](https://github.com/kaltura/playkit-ios/issues): ______
+- [ ] Which Player [version](https://github.com/kaltura/playkit-ios/releases) are you using: ______
+- [ ] Can you reproduce the issue with our latest release version: ______
+- [ ] Can you reproduce the issue with the latest code from master: ______
+- [ ] What devices and iOS versions are you using: ______
```

Comment:
Let's leave it OS instead of iOS so it will be generic across repos, I believe people will understand the context :-)